.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,9 @@
+### Commit Message
 
+COMMIT MESSAGE HERE (preserve newline after "Commit Message" heading)
+
+### End Commit Message
```

Comment:
something like this maybe?

```suggestion
### Commit Message
COMMIT MESSAGE SUBJECT LINE

COMMIT MESSAGE BODY
### End Commit Message
```

Reply:
See PR description, but the whole point is that you do not need the subject line...